client/src/pages/Register/PayPalButtons.tsx

```diff
@@ -35,6 +39,22 @@ interface InstanceData {
 };
 
 function PayPalButtonsComponent(props: Props) {
+  const [, payPalDispatch] = usePayPalScriptReducer();
+  const history = useHistory();
+
+  React.useEffect(() => {
+    payPalDispatch({
+      type: 'setLoadingStatus',
+      value: 'pending' as SCRIPT_LOADING_STATE,
+    });
+
+    // destroy on back, so this can be recreated
+    const listener = history.listen((location, action) => {
```

Comment:
There were orignally issues with the back/forward and loading the paypal script.  I found this solution in the paypal JS docs.  This creates a listener that "destroys" the paypal script component when you navigate away.  This means that the component can regenerate the script fresh when you navigate to it.